baseframe/forms/widgets.py

```diff
@@ -120,11 +120,11 @@ def __call__(self, field, **kwargs):
             value = ' '
         class_ = kwargs.pop('class', kwargs.pop('class_', ''))
         date_value, time_value = value.split(' ', 1)
-        return Markup(u'<input type="date" class="datetime-date %s" %s /> <input type="text" class="datetime-time %s" %s /> %s' % (
+        return Markup(u'<input type="date" class="datetime-date %s" %s /> <input type="time" class="datetime-time %s" %s /> %s' % (
             class_,
             wtforms.widgets.html_params(name=field.name, id=field_id + '-date', value=date_value, **kwargs),
             class_,
-            wtforms.widgets.html_params(name=field.name, id=field_id + '-time', value=time_value, **kwargs),
+            wtforms.widgets.html_params(name=field.name, id=field_id + '-time', value=time_value, placeholder=_("--:--"), **kwargs),
```

Comment:
`_()` is not required here as far as I can tell.

Reply:
Fixed